packages/flutter_tools/lib/src/asset.dart

```diff
@@ -435,6 +483,14 @@ Map<_Asset, List<_Asset>> _parseAssets(
   return result;
 }
 
+_Asset _resolvePackageAsset(
+    String assetBase, String packageName, String asset) {
```

Comment:
I'm fine with adding trailing `,` but this file does not seem to adopt that convention...